### PR TITLE
i18n(hi): add `customization.mdx`

### PR DESCRIPTION
**Pull request by at-the-vr**
#### Description
- adds translation for [Customizing Starlight](https://starlight.astro.build/hi/guides/customization/) in hindi language

### CODE REVIEW

File: docs/src/content/docs/hi/guides/customization.mdx [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+   - src/
+     - content/
+     - fonts/
+       - **CustomFont.woff2**
+       - **font-face.css**
+   - astro.config.mjs
+
+   </FileTree>
+
+2. `src/fonts/ में अपने प्रत्येक फ़ॉन्ट के लिए एक [`@font-face` घोषणा](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face) जोड़ें फ़ॉन्ट-face.css`।
```
Comment (HiDeoo) [inline docs/src/content/docs/hi/guides/customization.mdx:326]: Looks like some formatting got mixed up here, the `src/fonts/font-face.css` path is incomplete and I think the inline code is missing its termination mark.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the new translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I just left 1 comment for a markup issue but besides that, everything looks perfect to me :thumbsup:

File: docs/src/content/docs/hi/guides/customization.mdx (hunk 1 above)
Comment (at-the-vr) [inline docs/src/content/docs/hi/guides/customization.mdx:326]:
```suggestion
2. `src/fonts/font-face.css` में अपने प्रत्येक फ़ॉन्ट के लिए एक [`@font-face` घोषणा](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face) जोड़ें।
```

**Review by HiDeoo — APPROVED** [review]
Thanks for the quick fix 🙌

**Pull request merged**